### PR TITLE
[StructuralMechanicsApplication] Adding collapse option to ShellToSolidShellProcess

### PR DESCRIPTION
**Pull request by loumalouomega**
This allows to collapse a solid into a shell (basically the inverse operation of the current one)

### CODE REVIEW

**Comment by loumalouomega** [comment]
Ping

**Comment by loumalouomega** [comment]
Ping

**Comment by philbucher** [comment]
sorry I forgot
WIll try to review tmr

**Comment by loumalouomega** [comment]
Thanks

File: applications/StructuralMechanicsApplication/custom_processes/shell_to_solid_shell_process.cpp [hunk 1]
```diff
@@ -397,22 +632,22 @@ inline void ShellToSolidShellProcess<TNumNodes>::ComputeNodesMeanNormalModelPart
...
-        auto it_node = nodes_array.begin() + i;
+        auto it_node = r_nodes_array.begin() + i;
 
-        array_1d<double, 3>& normal = it_node->GetValue(NORMAL);
-        const double norm_normal = norm_2(normal);
-        if (norm_normal > tolerance) normal /= norm_normal;
+        array_1d<double, 3>& r_normal = it_node->GetValue(NORMAL);
+        const double norm_normal = norm_2(r_normal);
+        if (norm_normal > tolerance) r_normal /= norm_normal;
         else KRATOS_ERROR << "WARNING:: ZERO NORM NORMAL IN NODE: " << it_node->Id() << std::endl;
```
Comment (philbucher) [inline applications/StructuralMechanicsApplication/custom_processes/shell_to_solid_shell_process.cpp:651]:
```suggestion
         else KRATOS_ERROR << "ZERO NORM NORMAL IN NODE: " << it_node->Id() << std::endl;
```

**Review by philbucher — APPROVED** [review]
I am not the expert here but I don't see any obvious errors, therefore approving

maybe @RiccardoRossi has time to take a look, but if not feel free to go ahead with merging

File: applications/StructuralMechanicsApplication/custom_processes/shell_to_solid_shell_process.cpp (hunk 1 above)
Comment (loumalouomega) [reply applications/StructuralMechanicsApplication/custom_processes/shell_to_solid_shell_process.cpp:651]: I prefer to merge now, Travis is slow lately

**Pull request merged**